sql/moz-fx-data-shared-prod/subscription_platform_derived/stripe_subscriptions_changelog_v1/metadata.yaml

```diff
@@ -0,0 +1,25 @@
+friendly_name: Stripe subscriptions changelog
+description: |-
+  Changes to Stripe subscriptions as recorded by Fivetran (https://fivetran.com/docs/applications/stripe).
+  This table's schema closely mirrors Stripe's subscriptions API (https://stripe.com/docs/api/subscriptions/object).
```

Comment:
I wanted to keep the table description focused on only the most relevant details, so I didn't include the full context for the design decision to have the table mirror the Stripe subscriptions API, but the major reason for that is I expect us to replace Fivetran as the Stripe subscriptions data source with either the SubPlat subscriptions Firestore collection changelog sync or a custom sync of subscription create/update/delete events from Stripe's API, both of which would have Stripe subscription data in the API format.

And the major reason for that expectation is that if/when SubPlat allows Stripe subscriptions with multiple subscription items, Fivetran's `subscription_history` CDC sync will no longer be sufficient to track all subscription plan changes.

Also, the potential for Fivetran to lose `subscription_history` data (which they're refusing to fix) is not an ideal situation to remain in.